README.md

```diff
@@ -479,52 +479,56 @@ performance.
 
 ##### Decoupled Responses
 
-It is also possible for a backend to send multiple responses for a
-request or not send any responses for a request. A backend may also
+It is also possible for a backend to send multiple responses
+for a request. A backend may also
 send responses out-of-order relative to the order that the request
 batches are executed. Such backends are called *decoupled* backends.
-The decoupled backends use one `ResponseFactory` object per request to keep
-creating and sending any number of responses for the request. For this
+
+The decoupled backends use one `ResponseFactory` object per request to
+create and send any number of responses for the request. They must send at
+least one final response per request (even if it contains no output tensors).
```

Comment:
```suggestion
least one final response per request (even if it is flags-only response).
```

Reply:
Also add a line saying `TRITONBACKEND_ResponseFactorySendFlags` can be used to send these flags-only response from the backend.

Reply:
@tanmayv25 I think the second part is handled in the steps outlined later?

Reply:
I was thinking of elaborating on what actually a response with no output tensors or flags-only empty response is.

Reply:
I updated the verbiage and added the clarifying sentence. Tanmay, let me know what you think.